de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/location/geocoding/openstreetmap/NominatimOSMGeographicalCoordinatesJSON.java

```diff
@@ -0,0 +1,34 @@
+package org.adempiere.location.geocoding.openstreetmap;
+
+import lombok.Data;
+
+/*
+ * #%L
+ * metasfresh-pharma
+ * %%
+ * Copyright (C) 2018 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+@Data
```

Comment:
can't we make this immutable, i.e. `@Value`?

Reply:
Done